src/api/routes/application.js

```diff
@@ -12,15 +11,15 @@ module.exports = (app) => {
     /**
      * Creates a new Company Application
      */
-    router.post("/", validators.create, async (req, res) => {
+    router.post("/", validators.create, async (req, res, next) => {
+
         try {
             // This is safe since the service is destructuring the passed object and the fields have been validated
             const application = await (new ApplicationService()).create(req.body);
 
             return res.json(application);
         } catch (err) {
-            console.error(err);
-            return res.status(HTTPStatus.INTERNAL_SERVER_ERROR).send();
+            return next(err);
```

Comment:
Can you please substitute the catch in the offer endpoints (there are only a few) that are doing the error handling